frontend/src/libraries/explorer-wamp/stats.ts

```diff
@@ -6,9 +6,9 @@ export interface TransactionsByDate {
   transactionsCount: number;
 }
 
-export interface TeragasUsedByDate {
+export interface GasUsedByDate {
   date: string;
-  teragasUsed: number;
+  gasUsed: number;
```

Comment:
Is it a number or a string?

Reply:
Thank you for the catch! Much better to use string here, this number does not fit into JS int